shell/platform/linux/fl_backing_store_provider.cc

```diff
@@ -78,7 +78,11 @@ uint32_t fl_backing_store_provider_get_gl_target(FlBackingStoreProvider* self) {
 }
 
 uint32_t fl_backing_store_provider_get_gl_format(FlBackingStoreProvider* self) {
-  return GL_RGBA8;
+  if (epoxy_is_desktop_gl()) {
```

Comment:
Why are the formats different? This needs a comment to explain why each format is chosen.

Reply:
Done.

Reply:
But the reason is not explained in Skia. Comments here will only reference Skia code.

Reply:
That's fine, referring to Skia at least leaves a trail to find the root cause.